docs/registry/upload/_index.md

```diff
@@ -289,8 +306,8 @@ However, if you already have your own CI with access to arm runners or only inte
 
 1. Paste one of the following action templates into the edit window, depending on whether you are using the `build-action` or `upload-module` action:
 
-   {{< tabs >}}
-   {{% tab name="CI with build-action" %}}
+{{< tabs >}}
```

Comment:
@andf-viam do you have any idea why it keeps telling me this tab is unclosed and failing to build on hugo when AFAIK it is closed? I keep checking and I can't find where I changed the tabs to be wrong but they are messed up